experiments/AMIP/components/land/bucket_init.jl

```diff
@@ -226,26 +266,19 @@ function bucket_init(
 
     # Initial conditions with no moisture
     Y, p, coords = initialize(model)
-    Y.bucket.T = map(coords.subsurface) do coord
-        T_sfc_0 = FT(271.0)
-        radlat = coord.lat / FT(180) * pi
-        ΔT = FT(0)
-        if land_temperature_anomaly == "zonally_asymmetric"
-            anom_ampl = FT(0)# this is zero, no anomaly
-            lat_0 = FT(60) / FT(180) * pi
-            lon_0 = FT(-90) / FT(180) * pi
-            radlon = coord.long / FT(180) * pi
-            stdev = FT(5) / FT(180) * pi
-            ΔT = anom_ampl * exp(-((radlat - lat_0)^2 / 2stdev^2 + (radlon - lon_0)^2 / 2stdev^2))
-        elseif land_temperature_anomaly == "aquaplanet"
-            ΔT = FT(29) * exp(-coord.lat^2 / (2 * 26^2))
-        elseif land_temperature_anomaly == "amip"
-            ΔT = FT(40 * cos(radlat)^4)
-        else
-            ΔT = FT(0)
-        end
-        T_sfc_0 + ΔT
-    end
+
+    # Get temperature anomaly function
+    T_functions = Dict(
+        "aquaplanet" => temp_anomaly_aquaplanet,
+        "amip" => temp_anomaly_amip,
+        "zonally_asymmetric" => temp_anomaly_zonally_asymmetric,
+    )
+    haskey(T_functions, land_temperature_anomaly) || error("land temp anomaly function not supported")
+    temp_anomaly = T_functions[land_temperature_anomaly]
+
+    # Set temperature IC including anomaly
+    T_sfc_0 = FT(271.0)
```

Comment:
Where does this (and the other) numbers come from?

Reply:
Also, I think you don't need `FT` here.

Reply:
This follows the atmos setup [here](https://github.com/CliMA/ClimaAtmos.jl/blob/85467c4d3013c3495e925f8952971012f67fb6ad/src/initial_conditions/atmos_state.jl#L73).